src/ephy-bookmark.c

```diff
@@ -342,3 +485,72 @@ ephy_bookmark_tags_compare (const char *tag1, const char *tag2)
 
   return result;
 }
+
+char *
+ephy_bookmark_to_bso (EphyBookmark *self)
```

Comment:
@mcatanzaro `BSO` stands for [Basic Store Object](https://docs.services.mozilla.com/storage/apis-1.5.html#basic-storage-object) - a generic JSON wrapper around all items passed into and out of the SyncStorage server.